src/Storages/StorageMongoDBSocketFactory.cpp

```diff
@@ -0,0 +1,64 @@
+#include "StorageMongoDBSocketFactory.h"
+
+#include <Common/Exception.h>
+
+#if !defined(ARCADIA_BUILD)
+#   include "config_core.h"
+#   include <Common/config.h>
+#endif
+
+#include <Poco/Net/IPAddress.h>
+#include <Poco/Net/SocketAddress.h>
+
+#if USE_SSL
+#   include <Poco/Net/SecureStreamSocket.h>
+#endif
+
+#ifdef __clang__
+#   pragma clang diagnostic ignored "-Wunused-parameter"
+#endif
```

Comment:
May be better add for arguments in `createSecureSocket` an attribute [[maybe_unused]]

Reply:
Thanks a lot! I didn't know `[[maybe_unused]]` actually exists. 😀